src/attendee/Attendee.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[Attendee]] contains the information of an attendee.
+ */
+export default class Attendee {
+  /**
+   * The attendee id of an attendee.
+   */
+  attendeeId: string;
+  /**
+   * The external user id of an attendee.
+   */
+  externalUserId?: string;
```

Comment:
`externalUserId` is a required field when the builder creates an attendee using AWS SDK, REST API, or CLI. (We made `externalUserId` optional in some places as we added it in a later version.)
https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html#API_CreateAttendee_RequestSyntax